diff --git a/arch/arm/mach-omap2/Makefile b/arch/arm/mach-omap2/Makefile
index 512b152..fdc2e45 100644
--- a/arch/arm/mach-omap2/Makefile
+++ b/arch/arm/mach-omap2/Makefile
@@ -179,6 +179,7 @@ obj-$(CONFIG_MACH_OMAP_2430SDP)             += board-2430sdp.o \
                                           hsmmc.o
 obj-$(CONFIG_MACH_OMAP_APOLLON)                += board-apollon.o
 obj-$(CONFIG_MACH_OMAP3_BEAGLE)                += board-omap3beagle.o \
+             board-omap3beagle-camera.o \
                                           hsmmc.o
 obj-$(CONFIG_MACH_DEVKIT8000)          += board-devkit8000.o \
                                            hsmmc.o
@@ -190,6 +191,7 @@ obj-$(CONFIG_MACH_OMAP3530_LV_SOM)      += board-omap3logic.o \
 obj-$(CONFIG_MACH_OMAP3_TORPEDO)        += board-omap3logic.o \
                                           hsmmc.o
 obj-$(CONFIG_MACH_OVERO)               += board-overo.o \
+             board-overo-eslCam.o \
                                           hsmmc.o
 obj-$(CONFIG_MACH_OMAP3EVM)            += board-omap3evm.o \
                                           hsmmc.o


